carbonmark/components/pages/Project/Retire/Pool/TotalValues.tsx

```diff
@@ -76,6 +79,14 @@ export const TotalValues: FC<TotalValuesProps> = (props) => {
     selectiveFee();
   }, []);
 
+  /** Surface form errors */
+  useEffect(() => {
+    const error = Object.values(formState.errors).at(0);
+    if (notNil(error)) {
+      setError(error.message ?? "");
+    }
+  }, [formState.errors]);
```

Comment:
I know this is not your doing but, why do we even need to set the error.message to state, can't we just render the message right in jsx? useFormContext should trigger the necessary component re-render.